src/Microsoft.PowerShell.Commands.Utility/commands/utility/UtilityCommon.cs

```diff
@@ -238,16 +238,19 @@ public override string ToString()
                     // Display each byte, in 2-digit hexadecimal, and add that to the left-hand side.
                     nextLine.AppendFormat("{0:X2} ", currentByte);
 
-                    // If the character is printable, add its ascii representation to
-                    // the right-hand side.  Otherwise, add a dot to the right hand side.
-                    if ((currentByte >= 0x20) && (currentByte <= 0xFE))
-                    {
-                        asciiEnd.Append((char)currentByte);
-                    }
-                    else
-                    {
-                        asciiEnd.Append('.');
+                    // Add its ascii representation to the right-hand side.
+                    // If the byte represents a control character, replace with a space for byte value 0   
+                    // and /uFFFD for all others.
+                    char charToAppend = (char)currentByte;
+                    if (char.IsControl(charToAppend)) {
+                        // 
+                        if (currentByte == 0) {
+                            charToAppend = ' ';
```

Comment:
should this be a space or the null replacement character?

Reply:
This is a space - see 'https://github.com/PowerShell/PowerShell/issues/7777#issuecomment-498863328'

Reply:
Technically this is a null.  How would I tell the difference between a null and a space?

Reply:
@TravisEz13 - In this regard I think there's little difference between what the present build does (replacing control chars with a period). Having NUL replaced by a space (rather than another replacement character) imo makes it easier to read Unicode bytes converted in this way (see screenshots in previous comment link). 

I'm happy to change this behaviour and replace NUL with /u2400 or /uFFFD. If we do, and go /u2400 then maybe replace other control characters with their Unicode symbol equivalents as well (I'm not convinced doing that would be efficient or effective for this method).

Reply:
Yes, if /u2400, is replaced, it will be replaced with the same character as /uFFFD.

Reply:
I agree your behavior is net better than the current behavior, so since you have thought about this.  You can resolve this comment.

Reply:
I'll leave my feedback as request changes until you address the test failures.

Reply:
@TravisEz13 Thank you. On the subject of Unicode/test failures - the test is failing because of a case sensitivity check - it isn't expecting the Unicode replacement charcter. The Format-Hex.Tests.ps1 file has the ExpectedResult on line 88, but I can't update it (if I should) with the correct Unicode character without changing the encoding of the .ps1 file.